deployment_scripts/config

```diff
@@ -18,8 +18,4 @@ WINDOWS_64_BINARY_NAME="monkey-windows-64.exe"
 TRACEROUTE_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute64"
 TRACEROUTE_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute32"
 SAMBACRY_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/sc_monkey_runner64.so"
-SAMBACRY_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/sc_monkey_runner32.so"
-
-# Mongo url's
-MONGO_DEBIAN_URL="https://downloads.mongodb.org/linux/mongodb-linux-x86_64-debian81-latest.tgz"
-MONGO_UBUNTU_URL="https://downloads.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-latest.tgz"
```

Comment:
Where did this go? Don't we want to have a single file with all links as a single source of truth to edit?

Reply:
It's part of install_mongo.sh

Reply:
It anyway wasn't updated

Reply:
It might be beyond this PR, but why do we need a separate install_mongo.sh? If it's to split up deployment script into smaller, more manageable then let's force all subscripts to use the same config file.

Reply:
Unless the configs are interdependent what's the point?
Also, between linux and windows there are differences in what you can assume as programs.
In Windows it's easy to make a JSON file and parse it. Linux, easier to parse a simple text file.

I don't see any advantage to combining them. Not like we have an installer.

Reply:
At no point I suggested "lets make linux and windows use the same config file", I was talking about `install_mongo.sh` script. As it is right now we have to update links in `config` and script in `deploy_linux.sh`. As it stands it's easy to update, if you want to update version you update links, if you want to update logic you update the actual deployment script. But then there's also a separate `install_mongo.sh` script that has it's own links and logic and also has to be maintained. Why this particular part of deployment is separated? this separation means that you can't just review the config file and be sure that everything is up to date.